scripts/docker/builder/init.Dockerfile

```diff
@@ -6,5 +6,3 @@ RUN curl https://sh.rustup.rs -sSf | bash -s -- -y
 ENV PATH="/root/.cargo/bin:${PATH}"
 RUN rustup update nightly
 RUN rustup target add wasm32-unknown-unknown --toolchain nightly
-RUN rustup update stable
-RUN cargo install --git https://github.com/alexcrichton/wasm-gc
```

Comment:
```RUN cargo install --git https://github.com/alexcrichton/wasm-gc```

Not sure this is not needed anymore?

Reply:
Yes, in substrate 2.0 we don't need it anymore.